exercises/go-counting/src/test/java/GoCountingTest.java

```diff
@@ -0,0 +1,175 @@
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.awt.Point;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Set;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Rule;
+import org.junit.rules.ExpectedException;
+
+import java.util.HashSet;
+
+public class GoCountingTest {
+	
+	@Rule
+	public ExpectedException expectedException = ExpectedException.none(); 
+	
+	String board5x5 = "  B  \n" +
+	                  " B B \n" +
+	                  "B W B\n" +
+	                  " W W \n" +
+	                  "  W  ";
+	
+	@Test
+	public void blackCorner5x5BoardTest() {
+		GoCounting gocounting = new GoCounting(board5x5);
+		
+		Set<Point> territory = new HashSet<>();
+		territory.add(new Point(0, 0));
+		territory.add(new Point(0, 1));
+		territory.add(new Point(1, 0));
+		
+		assertEquals(gocounting.BLACK, gocounting.getTerritoryOwner(0, 1));
+		assertEquals(territory, gocounting.getTerritory(0, 1));
+	}
+	
+	@Test
```

Comment:
As described in our [policies doc](https://github.com/exercism/java/blob/master/POLICIES.md#ignore-noninitial-tests) all but the first test should be ignored 🙂